src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.html

```diff
@@ -259,8 +256,6 @@
                  name="path"
                  formControlName="path"
                  [ngbTypeahead]="bucketDataSource"
-                 (selectItem)="pathChangeHandler()"
```

Comment:
just to understand this better, why are we removing these?

Reply:
This function was removed from ts file so removing from html file.

Reply:
ah okay, that's good enough reason then